addon/tracked.ts

```diff
@@ -2,6 +2,10 @@ import Ember from 'ember';
 import { assert } from '@ember/debug';
 import { DEBUG } from '@glimmer/env';
 import { addObserver } from '@ember/object/observers';
+import { decoratorWithParams } from '@ember-decorators/utils/decorator';
+
+// @ts-ignore
+const notifyPropertyChange: (target: object, key: string) => void = Ember.notifyPropertyChange;
```

Comment:
it's missing from the types, likely because I couldn't get a straight answer around public API of `import Ember from 'ember';`.

<img width="188" alt="screen shot 2018-12-11 at 7 41 31 pm" src="https://user-images.githubusercontent.com/558005/49845731-c94f7580-fd7c-11e8-9c53-2bd261968d2e.png">


Let's definitely not handle this with a ts-ignore, since it tosses a large portion of type safety. I'll try and make an upstream fix this week, and it'll probably be released on the same day

Reply:
Heh. Hold yer horses there 🐎. The last time you brought this up, I fixed the docs: 
https://github.com/emberjs/ember.js/pull/16984. This is included in 3.6, and should show up in emberjs.com/api/ when the learning team deploys the 3.6 docs.

Reply:
@rwjblue and I have queued up 
- [x] the appropriate re-export of this through `@ember/object` https://github.com/ember-cli/ember-rfc176-data/pull/84
- [x] changes to the docs https://github.com/emberjs/ember.js/pull/17357
- [x] new versions of ember-cli-babel 6, 7 - https://github.com/emberjs/ember.js/pull/17357#issuecomment-446739150
- [ ] changes to the types https://github.com/DefinitelyTyped/DefinitelyTyped/pull/31333

Reply:
@rwjblue RE: fixing the docs in general, I'm referring to the fact that the `ember` module is not represented as public in the docs, and thus I have no idea what it means for the contents of a non-public module to be public.

Sometimes (not from you) I get a response like "people should just look at the ember 2.16 docs". That's about as much info as I've been able to obtain, and still sounds like an unaccetable solution to me

Reply:
> people should just look at the ember 2.16 docs

WAT!!!!!?!?!?!?!?! That is bonkers (and horrible) 😭

Reply:
UPDATE: most recently published versions of types for `ember` and `@ember/object` now contain `notifyPropertyChange`.

I suggest

```ts
import { notifyPropertyChange } from '@ember/object'
```